python/tvm/relay/backend/contrib/ethosu/tir/dma.py

```diff
@@ -197,18 +197,72 @@ def get_read_params(stmt):
 
     base_address = [get_base_address(index) for index in inner.value.indices]
     data_type = inner.buffer.data.type_annotation.element_type.dtype
+
+    def check_rolling_buffer():
```

Comment:
Lets make this a function of dma.py and make it accept stmt which is a read loop nest.

Reply:
Done